mantle/cmd/kola/testiso.go

```diff
@@ -336,6 +336,10 @@ func runTestIso(cmd *cobra.Command, args []string) error {
 		if kola.CosaBuild.Meta.BuildArtifacts.LiveIso == nil {
 			return fmt.Errorf("build %s has no live ISO", kola.CosaBuild.Meta.Name)
 		}
+		if strings.Contains(kola.CosaBuild.Meta.BuildID, ".dev.") {
+			// Default to ignore signing verification when running with development build
+			baseInst.Insecure = true
```

Comment:
This is modifying the base installation object. Which I think is correct, but it should be done upfront instead of as part of the `iso-install` scenario. E.g. I think we want this for the PXE case too.

So basically, you can move this hunk to around line 303 where we handle the insecure flag already. Then you can also remove the "for iso-install" bit from the commit message title.

Reply:
> This is modifying the base installation object. Which I think is correct, but it should be done upfront instead of as part of the `iso-install` scenario. E.g. I think we want this for the PXE case too.

Indeed. I didn't test the `pxe-install` case locally (with a dev build), but it also has the same problem so we'd want to try to fix it for both as you suggest.